src/main/java/seedu/linkedout/logic/commands/EditCommand.java

```diff
@@ -103,7 +104,9 @@ private static Applicant createEditedApplicant(Applicant applicantToEdit,
         Job updatedJob = editApplicantDescriptor.getJob().orElse(applicantToEdit.getJob());
         Round updatedRound = editApplicantDescriptor.getRound().orElse(applicantToEdit.getRound());
         Set<Skill> updatedSkills = editApplicantDescriptor.getSkills().orElse(applicantToEdit.getSkills());
-        return new Applicant(updatedName, updatedPhone, updatedEmail, updatedJob, updatedRound, updatedSkills);
+        Flag updatedFlag = applicantToEdit.getFlag();
+        return new Applicant(updatedName, updatedPhone, updatedEmail, updatedJob, updatedRound, updatedSkills,
+                updatedFlag);
```

Comment:
Implementation looks sound, just need to clarify whether there might be any case where applicantToEdit flagged is able to be set to null

Reply:
Looks good, maybe rename updateFlag to just flag to let the dev reading know that flag is a copy of the original and not an updated value.

Reply:
Changed!